bindings/twilio/sms.go

```diff
@@ -69,20 +70,27 @@ func (t *SMS) Init(metadata bindings.Metadata) error {
 	}
 
 	t.metadata = twilioM
+	t.httpClient.Timeout = twilioM.timeout
+
 	return nil
 }
 
 func (t *SMS) Write(req *bindings.WriteRequest) error {
+	toNumberValue := t.metadata.toNumber
+	if toNumberValue == "" {
+		toNumberFromRequest, ok := req.Metadata[toNumber]
+		if !ok || toNumberFromRequest == "" {
+			return errors.New("twillio missing \"to\"")
```

Comment:
typo, `twillio` should be `twilio`. Better to rephrase as `twilio missing toNumber field`.